server.py

```diff
@@ -0,0 +1,63 @@
+from flask import Flask, jsonify, request, abort
+from models import db, ACMRequest
+from settings import MYSQL
+from sqlalchemy.sql.expression import func, text
+from datetime import datetime
+import logging
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://{}:{}@{}/{}'.format(
+    MYSQL['user'],
+    MYSQL['password'],
+    MYSQL['host'],
+    MYSQL['dbname']
+)
+
+db.init_app(app)
+db.create_all(app=app)
+
+@app.route('/request', methods=['GET'])
+def get_all_acm_requests():
+    acm_requests = [r.to_dict() for r in ACMRequest.query.all()]
```

Comment:
It might make sense to only return uncompleted requests. As we collect more requests over time, we don't want to send every request ever made in the response.

Reply:
That makes sense. Just pushed up the change. Should we still later add a way of accessing the completed requests?